src/qforte/adapters/molecule_adapters.py

```diff
@@ -235,6 +235,27 @@ def create_external_mol(**kwargs):
     with open(kwargs["filename"]) as f:
         external_data = json.load(f)
 
+    # extract symmetry information if found
+    try:
+        point_group = external_data['point_group']['data']
+    except:
```

Comment:
Bare except statements are dangerous! Please include the specific exception being caught. (Probably `KeyError`.)